script/profile_db_generator.rb

```diff
@@ -65,7 +65,7 @@ def create_admin(seq)
 end
 
 def ensure_perf_test_topic_has_right_title!
-  t = Topic.find(179)
+  t = Topic.where(archetype: :regular).last
```

Comment:
minor but I think this picks an inconsistent topic, topics have random numbers of posts so we really want to pick a topic with exactly N posts in it. will adjust.